ramlfications/errors.py

```diff
@@ -31,6 +31,19 @@ class InvalidSecuritySchemeError(InvalidRAMLError):
     pass
 
 
+class MultipleValidationErrors(Exception):
+    def __init__(self, errors):
+        super(MultipleValidationErrors, self).__init__(
+            "Multiple validation errors were found.")
+        self.errors = errors
+
+    def __str__(self):
+        output = "\n"
+        for e in self.errors:
+            output += "{0}: {1}\n".format(e.__class__.__name__, e)
```

Comment:
What do you thinking of adding a `\t` (tab) when printing, at least to the console?

```python
MultipleValidationErrors:
    InvalidRootNodeError: RAML File does not define the baseUri.
    InvalidRootNodeError: RAML File does not define an API title.
```